### PR TITLE
Docs: Ensure the GitHub link to the source is formed correctly

### PR DESCRIPTION
**Pull request by pierlon**
## Summary

This PR updates how the the direct link to the source in the repository is formed. This should fix any broken links that may be found currently within the docs.

## Checklist

- [ ] My pull request is addressing an open issue (please create one otherwise).
- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: docs/src/Model/HasCodeLinks.php
Comment (pierlon) [inline docs/src/Model/HasCodeLinks.php:39]: @schlessera would this todo still be relevant with the change below? The link would be relevant to the branch or tag being browsed.

**Comment by pierlon** [comment]
Travis build failure is not related to the changes here.

**Comment by westonruter** [comment]
Please rebase to fix the Travis build error.

**Comment by pierlon** [comment]
Rebased.

**Review by westonruter — APPROVED** [review]
Changes look good, but I defer to @schlessera.

File: docs/src/Model/HasCodeLinks.php
Comment (schlessera) [reply docs/src/Model/HasCodeLinks.php:39]: No, this is not relevant anymore.

**Comment by schlessera** [comment]
@pierlon Can you post such a broken link into the issue description for reference? I'm curious, as the ones I tested did work (and no, I did not test _all_ of them  😜).

**Comment by schlessera** [comment]
Ah, nevermind. I saw the link from Alberto, and it turns out the link is only broken if you _don't_ use a direct hash or branch to refer to it. So you cannot actually reference a broken link in the description here that wouldn't be automatically fixed once the PR is merged.

**Comment by westonruter** [comment]
I'll take that as a ✅.

**Comment by westonruter** [comment]
I squashed 0aaa0cbb3c4be542dde8bd986703fcf3e3713b4f with a fresh regeneration of the docs from the `2.0` branch in 2c90061.

**Comment by schlessera** [comment]
**QA passed**

For reference, the link that was initially broken was the source link at the bottom of this page: https://github.com/ampproject/amp-wp/blob/develop/docs/method/AMP_Base_Sanitizer/sanitize.md

It works correctly now.

**Pull request merged**
